src/components/questionary/questionaryComponents/Interval/QuestionaryComponentInterval.tsx

```diff
@@ -132,6 +132,7 @@ export function QuestionaryComponentInterval(props: BasicComponentProps) {
             label="Min"
             id={`${id}-Min`}
             onChange={(event) => {
+              // eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain
```

Comment:
Not sure if you need this comment at all because I don't see any non-null assertion.